CHANGELOG/CHANGELOG-3.3.md

```diff
@@ -0,0 +1,210 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
+# Changelog since v3.2.0
+
+## Changes by Kind
+
+### Feature
+
+- Add command line arguments to configure leader election options ([#313](https://github.com/kubernetes-csi/external-attacher/pull/313), [@RaunakShah](https://github.com/RaunakShah))
+- Adds mappings for PV access modes to new CSI access modes: `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER`. ([#308](https://github.com/kubernetes-csi/external-attacher/pull/308), [@chrishenzie](https://github.com/chrishenzie))
+- Updates Kubernetes dependencies to v1.22.0 ([#321](https://github.com/kubernetes-csi/external-attacher/pull/321), [@chrishenzie](https://github.com/chrishenzie)) [SIG Storage]
+
+### Bug or Regression
```

Comment:
Do you also want to mention https://github.com/kubernetes-csi/external-attacher/pull/318?

Reply:
Added